### DISCUSSION TITLE
v3.1.4

### DISCUSSION THREAD

**Discussion by osy** [osy86/HaC-Mini#966] [Releases]
## Changes
* Fix SD card reader on macOS Monterey (#613)
* Fix APFS boot not found on macOS Mojave and Catalina (#939)

## Known Issues
* LAN2 is not working on macOS Monterey (#947). This requires a driver update from Small Tree.

## Messages
* If you are upgrading from v2.17 and below, please read the notes at the top of [v3.0.1](https://github.com/osy/HaC-Mini/releases/tag/v3.0.1)
* [As announced a few months ago](https://github.com/osy/HaC-Mini/discussions/578), this project will be transitioning to a long-term support model (a few stability updates per year). Following Apple's own deprecation model, we expect the EOL of this project to be no earlier than 2023.

<hr /><em>This discussion was created from the release <a href='https://github.com/osy/HaC-Mini/releases/tag/v3.1.4'>v3.1.4</a>.</em>

**Comment by controlfix87** [comment]
Hi @osy , 
You are writing that LAN2 is broken on Monterey.. is that mean that LAN1 is working? (as we have 2 ports there)?

Thanks :)

**Reply by osy** [reply]
Yes that’s why it says LAN “2”

**Reply by controlfix87** [reply]
LOL just double checked because I've tried to connect the LAN cable to LAN1 but no luck as well. 
Any ideas? 
Trying to avoid from downgrading to BigSur. 

Also both of the ports are enabled in BIOS

**Reply by osy** [reply]
Make sure it works in windows/Linux or you have a network issue.

**Reply by controlfix87** [reply]
It does work in windows 11 perfectly. But in MacOs the cable is connected but showing as disconnected. 
I will try again later today and update. 

Thank you ao much for your professional hard work :)

**Comment by alenzhong** [comment]
Airdrop from computer to iPad is invalid!
The iPad can be delivered to the computer by airdrop. 

1820A

**Comment by Oosram** [comment]
front blue USB port is not working on macOS Monterey, HacMini V3.1.4

**Reply by Oosram** [reply]
after restart，  it's OK now

**Comment by Maijestro** [comment]
No App Store Updates here….HacMini v3.1.4 i can Not Update to Monterey 12.2.

Can anybody help with this Problem ?

**Reply by Donan123** [reply]
I have the same issue with HacMini v3.1.4. I can’t update to Monterey 12.2 or 12.2.1 from 12.1.

**Reply by jisuye** [reply]
OTA to Monterey 12.2 is need Apple Secure Boot. Read This: [https://dortania.github.io/OpenCore-Post-Install/universal/security/applesecureboot.html](https://dortania.github.io/OpenCore-Post-Install/universal/security/applesecureboot.html)

**Reply by jisuye** [reply]
![image](https://user-images.githubusercontent.com/31565431/156116608-56976515-582f-4733-a021-c3102054dac1.png)

**Reply by jisuye** [reply]
Please note that the versions of OpenCore and OpenCoreConfigurator should be the same, otherwise the startup may fail after saving the configuration file. Please note to save the configuration of EFI directory to prevent startup failure.

**Reply by osy** [reply]
In HaC mini secure boot is one of the options you can check on install. But be warned that existing installs must be reinstalled or use bless in recovery to boot.

**Comment by GREEN88888** [comment]
hello blogger，Under Montery12.2.1 version, HACMINI3.14 is installed,
1. Occasionally restart - black screen when booting up, unplug the power and then plug it in to solve
2. The Thunderbolt3 interface can be used normally when one device is plugged in. After connecting two hard drives at the same time, the computer freezes. When the Thunderbolt3 patch was installed at that time, it indicated that it failed.
3. After connecting two 4K monitors and installing the anti-flicker patch, there is still a snowflake phenomenon.
4. The BCM94352Z wireless network card is used, do you need to replace the driver, because there is a stuck phenomenon when watching videos online, and browsing the web is normal.
